packages/loader/driver-definitions/src/driverError.ts

```diff
@@ -70,6 +70,8 @@ export enum DriverErrorType {
      * When it's hit, client needs to forget all the knowlege about this file and start over.
      */
      fileOverwrittenInStorage = "fileOverwrittenInStorage",
+
+     sessionForbidden = "sessionForbidden",
```

Comment:
Can you please add some description above it?
Also maybe clarify in name what "session" it refers to? I.e. deltaStreamSessionForbidden, or some wording that tells that it's fixed in time (StorageOnlyAllowed?)

Reply:
joins session is and odsp concept, the runtime only has connectToDeltaStream, so i think a better name would be deltaStreamConnectionForbidden